client/landing/stepper/declarative-flow/internals/steps-repository/assign-migration-trial-plan/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { PLAN_MIGRATION_TRIAL_MONTHLY } from '@automattic/calypso-products';
```

Comment:
We are prefix all site-migration related flows with `site-migration-*`
I am not a big fan of it and I don't have strong args, so feel free to keep the name if you have strong opinions about.